src/SourceBuild/content/repo-projects/vstest.proj

```diff
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <DeterministicBuildOptOut>true</DeterministicBuildOptOut>
+    <DoNotBuildTests>true</DoNotBuildTests>
```

Comment:
Should we file tracking issues for these exclusions and add comments here?

Reply:
Sure. I'll do that - thanks!

Reply:
Fixed with https://github.com/dotnet/sdk/pull/46394/commits/51744b8f0db7a7e446fdf678cf161a74607929c7